### PR TITLE
simple File Uploader

### PR DESCRIPTION
**Pull request by Gurusewak13**
+ Added style.css to define size of image to be uploaded
+FileUpload.jsx which implements file upload functionality

NOTE:

++ Currently works with images only ++

### CODE REVIEW

File: src/components/FileUpload.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+/**
+ *This is currently working with image files
+ */
+function FileUpload() {
+  // This stores the current state of the file
+  const [file, setFile] = React.useState("");
```
Comment (idomo1) [inline src/components/FileUpload.jsx:8]: Usually we import `useState` at the top
Change your import statement to `import React, { useState } from "react"`
Then this line can just be `... = useState("");`

File: src/components/App.jsx [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
-
+import FileUpload from './FileUpload'
 function App(){
     return(
         <Router>
             <Switch>
                 <Route exact path="/">
-                    <UploadPage />
+
+                    <FileUpload />
```
Comment (idomo1) [inline src/components/App.jsx:15]: The same with the other PR, this should be inside the UploadPage component

**Review by idomo1 — CHANGES_REQUESTED** [review]

File: src/components/FileUpload.jsx (hunk 1 above)
Comment (reteppeter) [reply src/components/FileUpload.jsx:8]: I don't think there is an issue with being explicit, especially when namespace access is only one level .

**Comment by mt-empty** [comment]
I didn't know you guys didn't have a `developer` branch, please change your target branch to the `master` branch.

**Review by mt-empty — CHANGES_REQUESTED** [review]
I didn't know you guys didn't have a `developer` branch, please change your target branch to the `master` branch.

**Comment by idomo1** [comment]
Superseded by this PR https://github.com/Visual-Testing-for-Android-Apps/web/pull/15

**Pull request closed without merging**